Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
It's consistent with every other usage of `xz` for LZMA in Homebrew's core, where there's an explicitly defined `option`.